htmd/membranebuilder/ringpenetration.py

```diff
@@ -96,10 +96,10 @@ def _checkAtomsPenetrating(coords2, bonds2, ringcoords, axis, ringcom, maxd, box
                 continue
 
             d = 0
-            for i in range(0, ringcoords.shape[0]):
-                p1 = ringcoords[i] - p
+            for k in range(0, ringcoords.shape[0]):
```

Comment:
ouch, this one is a classic :p

Reply:
yeah, I don't want to think how it worked ;P